src/test/lib/TestHelpers.h

```diff
@@ -319,14 +319,15 @@ struct StatelessRetryHelper
 
 #define PRIVATE_TP_TYPE   77
 #define PRIVATE_TP_LENGTH 2345
+#define PRIVATE_TP_LENGTH_HUGE 4134
 
 struct PrivateTransportHelper : QUIC_PRIVATE_TRANSPORT_PARAMETER
 {
-    PrivateTransportHelper(bool Enabled) {
+    PrivateTransportHelper(bool Enabled, bool Resumption = false) {
         if (Enabled) {
             Type = PRIVATE_TP_TYPE;
-            Length = PRIVATE_TP_LENGTH;
-            Buffer = new(std::nothrow) uint8_t[PRIVATE_TP_LENGTH];
+            Length = Resumption ? PRIVATE_TP_LENGTH : PRIVATE_TP_LENGTH_HUGE;
```

Comment:
Must do this because increasing the size increases the chance that 0-RTT doesn't work in the face of random packet loss.